test/neural_net/style_transfer.cxx

```diff
@@ -129,9 +245,57 @@ BOOST_AUTO_TEST_CASE(test_block_2) { style_transfer_test::test_block_2(); }
 
 BOOST_AUTO_TEST_CASE(test_vgg16) { style_transfer_test::test_vgg16(); }
 
-BOOST_AUTO_TEST_CASE(test_loss) { style_transfer_test::test_loss(); }
+BOOST_AUTO_TEST_CASE(test_pre_processing) {
+  style_transfer_test::test_pre_processing();
```

Comment:
Ignore some precedents you see in our code; there's no reason for this to call a separate `style_transfer_test::test_pre_processing()` function from here. Just define that function here. There's no reason to have every test named in two places

Reply:
Sounds good, it simplifies the code for sure.